.github/workflows/fetch_versions.yml

```diff
@@ -1,9 +1,12 @@
 name: Fetch latest versions
 
+# When triggering a manual execution of the workflow, you may want to disable the scheduled execution temporarily
+# to ensure that any in-flight generated PR is not overridden by the scheduled execution.
+
 on:
   schedule:
-    # Run once a day at 10 AM EST
-    - cron: "0 14 * * *"
+    # Run once a day at 8 AM EST
+    - cron: "0 12 * * *"
```

Comment:
I preferred this run during work hours in the morning since this is when we usually get to these issues.

Reply:
What's the rationale for changing this?

Reply:
You should receive notifications in Slack about the presence of the PR to remind you

Reply:
Typically this runs and alerts us when we are online, it's a good reminder to deal with this repository. Instead of dealing with the backlog of issues that accumulated.

@pkwarren and I get assigned (I can't speak for Philip) but I preferred this run when we're online instead of earlier in the day.